cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -21,6 +21,14 @@ var strings = require( '../modules/util/strings' );
  */
 function Multiselect( element ) { // eslint-disable-line max-statements, inline-comments, max-len
 
+  var BASE_CLASS = 'cf-multi-select';
+
+  // TODO: Add 'cf-multi-select' class to <select> menu markup
```

Comment:
Default multiselects are already styled (although very basically), so that's not an issue.

If we used a data-hook for the init, the styles would still only be attached to the class name, not the hook. The hook would only be for scripting, which is the intended use. As a side note, this component will support single option selects in the future using radio buttons instead of checkboxes, so naming will change at some point.